<!DOCTYPE html>
<html lang="en-US"><head>
		<meta charset="UTF-8">
		<title>FR940523-0-00040</title>
	</head>
	<body>
		<main>
			<p><!-- PJG STAG 4700 -->  <!-- PJG ITAG l=11 g=1 f=1 --> Regarding animal feed, food-producing animals consume primarily processed forms of cottonseed and rapeseed, in which, as discussed above, the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene is not expected to remain intact. In addition, researchers have shown that nucleic acids introduced into the rumens of calves, or incubated with calf, sheep, or cow rumen contents in vitro, were rapidly and completely degraded to nucleotides and nucleosides (Ref. 29).  <!-- PJG 0012 frnewline --> c.  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> Calculation of worst-case transformation frequencies.  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG 0012 frnewline --> In its submission, Calgene addressed the potential for horizontal transfer of the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene. Natural transformation, i.e., the uptake and incorporation into the genome of free DNA, is known to occur in some bacterial species. This is the only possible mechanism by which intestinal microflora could take up free DNA (Ref. 30). However, none of the species known to be present in the GI tract has been found capable of acquiring exogenous DNA by natural transformation. Nonetheless, to consider the worst-case scenario, Calgene assumed that all microbes in the intestine would be able to take up and incorporate exogenous DNA at a frequency found for certain species of the genus  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> Streptococcus.  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> Calgene noted that although the firm developed its transformation model for certain  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> Streptococcus  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> species, they are not aware of any information indicating that  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> Streptococcus <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  species found in the GI tract can be naturally transformed.  <!-- PJG 0012 frnewline --> To undergo natural transformation, the recipient bacterium must be transformation-competent, i.e., ready to take up DNA. As noted, none of the bacterial species that occur in the GI tract is known to be capable of becoming transformation-competent. In addition, the genome of a recipient bacterium should contain DNA homologous to the incoming DNA (Refs. 31 and 32). Because the genomes of intestinal  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> Streptococci  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> or other intestinal bacteria are not expected to exhibit homology to the DNA constructs containing the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> 6 <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> , Calgene assumed that the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene could only undergo ``illegitimate'' recombination, a process that does not require significant DNA homology. Calgene noted that illegitimate recombination occurs in microorganisms at a much lower rate than homologous recombination.  <!-- PJG 0012 frnewline -->  <!-- PJG 0012 frnewline -->  <!-- PJG /ITAG -->  <FOOTNOTE> <FOOTNAME> <!-- PJG ITAG l=28 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=5 f=1 --> 6 <!-- PJG /ITAG -->  </FOOTNAME> <!-- PJG ITAG l=28 g=1 f=1 --> One population that does contain DNA segments homologous with part of the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 -->  construct is  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> E. coli,  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 --> because the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 -->  construct contains part of an  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> E. coli  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 --> gene. Although  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> E. coli  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 --> constitutes one of the predominant species of aerobic GI tract bacteria,  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> E. coli  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 --> is not transformation-competent under conditions that prevail in the GI tract (Ref. 33). Thus, transformation of  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=3 --> E. coli  <!-- PJG /ITAG -->  <!-- PJG ITAG l=28 g=1 f=1 --> due to homologous recombination is not an issue.  <!-- PJG 0012 frnewline -->  <!-- PJG 0012 frnewline --> </FOOTNOTE>  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> Under the foregoing worst-case assumptions, Calgene estimated that if a person consumes fresh tomatoes at the 90th percentile level (i.e., eats more tomatoes than 89 percent of the individuals in the population), the transformation frequency of the intestinal microorganisms with the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene will be approximately 3&times;10 <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> ^15  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> transformants per day. This transformation frequency is more than 5 orders of magnitude less than the frequency of mutation to kanamycin resistance per bacterial replication, i.e., 10 <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> ^9 <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  (Ref. 12). Thus, Calgene showed that for every 300,000 bacteria that mutate to kanamycin resistance per replication (generally a matter of hours), there would be, at most, under worst-case conditions, one kanamycin-resistant bacterium per day added to that number due to transformation.  <!-- PJG 0012 frnewline --> Calgene stated that the potential for food-producing animals to experience decreased efficacy of antibiotic therapy as a result of pathogenic intestinal microflora incorporating and expressing the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene would be similar to that described for humans, i.e., equally improbable. In reaching this conclusion, Calgene relied on the finding that DNA is rapidly and completely digested in the gut of food animals (Ref. 29) and on the contention that the worst-case transformation scenario described above for human gut microorganisms also applies to microorganisms found in the gut of food-producing animals.  <!-- PJG 0012 frnewline --> With respect to epithelial cells lining the intestinal lumen, Calgene provided information that no transformation of human epithelial cells has been demonstrated in vivo (Ref. 2). In addition, even if transformed, intestinal epithelial cells are terminally differentiated (i.e., do not divide) and have a relatively short life span (Ref. 34), and thus would continually be shed and replaced by nontransformed cells.  <!-- PJG 0012 frnewline -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=81 g=1 f=1 --> 2. Potential Transfer of the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=81 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=81 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=81 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=81 g=1 f=1 -->  Gene to Soil Microorganisms  <!-- PJG 0012 frnewline -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 --> Calgene also considered the possibility that the  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=3 --> kan <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=5 f=1 --> r <!-- PJG /ITAG -->  <!-- PJG ITAG l=11 g=1 f=1 -->  gene might be transferred to soil microorganisms, thereby increasing the level of antibiotic-resistant organisms in the environment. Calgene pointed out that the only plausible mechanism by which gene transfer could occur between plants and bacteria is through natural transformation. Taking this mechanism into consideration and using worst-case assumptions similar to those discussed above for intestinal microorganisms, Calgene calculated that, at worst, kanamycin-resistant transformants resulting from plant DNA left in the fields would represent not more than one in 10 million of the existing kanamycin-resistant soil population.  <!-- PJG 0012 frnewline -->  <!-- PJG 0012 frnewline -->  <!-- PJG /ITAG -->  <!-- PJG /STAG --></p>
		</main>
</body></html>
            